addons/sale_pdf_quote_builder/tests/test_pdf_quote_builder.py

```diff
@@ -144,6 +145,24 @@ def test_dynamic_fields_mapping_for_product_document(self):
             )
             self.assertEqual(result, expected[form_field.name])
 
+    def test_product_document_dialog_params_access(self):
+        sale_order_user_demo = self.sale_order.copy({'user_id': self.user_demo.id})
+        self.user_demo.groups_id += self.env.ref('sales_team.group_sale_salesman')
```

Comment:
Truth be told, all users should have read access, but only read access :thinking:

Reply:
Yes, but why add the group salesman? It makes sense for the flow TBH, but we could test it with the user_demo without adding that group and it should still work

Reply:
Ohh! got it I though you meant user :slightly_frowning_face: